code/game/objects/items/weapons/cards_ids.dm

```diff
@@ -351,7 +351,7 @@
 						UpdateName()
 						to_chat(user, "Name changed to [new_name].")
 
-					if("Appearance")
+					if("Appearance") //what a snowflake fuck you
```

Comment:
You're not allowed to complain in the code in a small change. Fix or keep quiet.